whale/popup/popup.js

```diff
@@ -47,6 +47,26 @@ function openGitRepo() {
     let url = "https://github.com/seohyun0120/quick-sook";
     whale.tabs.create({ url });
 }
+function darkMode2(){
+    let body=document.querySelector('body');
+    let mode=document.querySelector('#dark_mode2');
+    let mode_img=document.querySelector('#mode_img')
+    if(mode.value==='bright')
+    {
+        body.style.backgroundColor="RGB(140,140,140)"
+        body.style.color='white';
+        mode.value="dark"
+        mode_img.src="../icons/moon.png"
+    }
+    else{
+        body.style.backgroundColor="white";
+        body.style.color="black";
+        mode.value= "bright"
+        mode_img.src="../icons/sun.png"
+    }
+
+}
+document.getElementById('dark_mode2').addEventListener('click', darkMode2);
 
```

Comment:
```suggestion
function darkMode2() {
    let body = document.querySelector('body');
    let mode = document.querySelector('#dark_mode2');
    let mode_img = document.querySelector('#mode_img');
    if (mode.value === "bright") {
        body.style.backgroundColor = "RGB(140,140,140)";
        body.style.color = 'white';
        mode.value = "dark";
        mode_img.src = "../icons/moon.png";
    } else {
        body.style.backgroundColor = "white";
        body.style.color = "black";
        mode.value = "bright";
        mode_img.src = "../icons/sun.png";
    }
}

document.getElementById('dark_mode2').addEventListener('click', darkMode2);
```

JS 코드 정렬 부탁드립니다 (_ _)